node.go

```diff
@@ -115,6 +111,11 @@ func (node *_Node) _GetChannel() (*_Channel, error) {
 					return nil
 				}
 
+				if !node.verifyCertificate {
+					println("skipping certificate check")
```

Comment:
Should move this logging statement to outside the callback so it's not called each time the connection is established.